sovrin_client/test/cli/test_pool_upgrade.py

```diff
@@ -110,4 +110,18 @@ def checksched():
                 assert node.upgrader.scheduledUpgrade
                 assert node.upgrader.scheduledUpgrade[0] == validUpgradeExpForceTrue[VERSION]
 
-    poolNodesStarted.looper.run(eventually(checksched, retryWait=1, timeout=10))
\ No newline at end of file
+    poolNodesStarted.looper.run(eventually(checksched, retryWait=1, timeout=10))
+
+
+def send_reinstall_true_upgrade_cmd(do, expect, upgrade_data):
+    do('send POOL_UPGRADE name={name} version={version} sha256={sha256} '
+       'action={action} schedule={schedule} timeout={timeout} reinstall=True',
+       within=10,
+       expect=expect, mapper=upgrade_data)
+
+
+def send_reinstall_false_upgrade_cmd(do, expect, upgrade_data):
+    do('send POOL_UPGRADE name={name} version={version} sha256={sha256} '
+       'action={action} schedule={schedule} timeout={timeout} reinstall=False',
+       within=10,
+       expect=expect, mapper=upgrade_data)
```

Comment:
The test looks pretty same as above. Do we show the same message in CLI in case of re-install True and False?
Can we check that Upgrade is actually scheduled?